tests/memory_checker/test_memory_checker.py

```diff
@@ -124,8 +123,8 @@ def restore_monit_config_files(duthost):
         None.
     """
     logger.info("Restoring original Monit configuration files on DuT '{}' ...".format(duthost.hostname))
-    duthost.shell("mv -f /tmp/monitrc /etc/monit/")
-    duthost.shell("mv -f /tmp/monit_* /etc/monit/conf.d/")
```

Comment:
why remove this?

Reply:
duthost.shell("mv -f /tmp/monit_* /etc/monit/conf.d/")

Reply:
backup section has

    duthost.shell("cp -rf /etc/monit /tmp/")

and then in restore its

    duthost.shell("rm -rf /etc/monit")
    duthost.shell("mv /tmp/monit /etc/monit")

Reply:
As Aaron mentioned the backup/restore code is still there it just uses a different approach.
Instead of only backing up some files under `/etc/monit` (which require due diligence as the test evolves), the test now backs up the entire content of `/etc/monit` which is much less error prone.